Library/Homebrew/tap.rb

```diff
@@ -127,12 +127,7 @@ def default_remote
     if OS.mac?
       "https://github.com/#{full_name}"
     else
-      case "#{user}/#{repo}"
-      when "Homebrew/dupes"
-        "https://github.com/Linuxbrew/homebrew-#{repo}"
-      else
-        "https://github.com/#{user}/homebrew-#{repo}"
-      end
+      "https://github.com/#{user}/homebrew-#{repo}"
```

Comment:
Change this to:
```
  def default_remote
    "https://github.com/#{full_name}"
  end
```

Reply:
@maxim-belkin Could you please make this change when you get to it?

Reply:
done!